app/components/owned-card-list.cjsx

```diff
@@ -5,14 +5,20 @@ Translate = require 'react-translate-component'
 apiClient = require 'panoptes-client/lib/api-client'
 PromiseRenderer = require '../components/promise-renderer'
 OwnedCard = require '../partials/owned-card'
-{Link} = require 'react-router'
+{Link, State, Navigation} = require 'react-router'
```

Comment:
I agree, these changes have broken the collections and favourites pages. I'm fairly sure we can revert any changes to this file.